### ISSUE TITLE
[v3] Implement new Concept Exercise: chars

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/fsharp#868]
This issue describes how to implement the `chars` concept exercise for the F# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of Chars in F#. We'll also use this exercise to introduce `for..in` loops and a mutable binding. 

## Learning objectives

- Know of the existence of the `char` type.
- Know what a char represents (a Unicode character).
- Know that chars are immutable.
- Know how to define a char.
- Know how to access a char in a string by index.
- Know of some basic char methods (like converting to uppercase).
- Know how to convert an integer to a character and vice versa.

## Out of scope

- Converting a string to a char array.

## Concepts

The Concepts this exercise unlocks are:

- `chars`: know of the existence of the `char` type; know what a char represents (a Unicode character); know that chars are immutable; know how to define a char; know how to access a char in a string by index; know of some basic char methods (like converting to uppercase); know how to convert an integer to a character and vice versa.

## Prerequisites

This exercise's prerequisites Concepts are:

- `strings`: know about working with strings.

### Hints

- [Char methods][char-methods]: lists the available methods on the `Char` class.

Once a reference document has been written on the subject of chars in F# (see section 11), it should be added too.

### After

- [Char methods][char-methods]: lists the available methods on the `Char` class.

Once a reference document has been written on the subject of chars in F# (see section 11), it should be added too.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/fsharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[how-to-implement-a-concept-exercise]: https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md
[implemented-exercises]: https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/README.md#implemented-exercises
[reference]: https://github.com/exercism/v3/blob/master/reference/types/char.md
[analyzer]: https://github.com/exercism/fsharp-analyzer
[representer]: https://github.com/exercism/fsharp-representer
[meta-design]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/design.md
[meta-config.json]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/config.json
[docs-concept-exercises]: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md
[char-methods]: https://docs.microsoft.com/en-us/dotnet/api/system.char?view=netcore-3.1#methods
